api/src/main/java/io/opentelemetry/trace/SemanticConventions.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.trace;
+
+/**
+ * Defines constants for all attribute names defined in the OpenTelemetry Semantic Conventions
+ * specifications.
+ *
+ * @see <a
+ *     href="https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/data-semantic-conventions.md">Semantic
+ *     Conventions</a>
+ */
+public final class SemanticConventions {
```

Comment:
Do we commit on these values to be 18-24 months backwards compatible? Maybe we should have a contrib package for this. 

@tedsuo @yurishkuro what is your experience with this?

Reply:
Ping @tedsuo and @yurishkuro

Reply:
I think we should start treating semantic conventions more like an API definition (a contract between instrumentation writers and the back-end), so I'd love if we could guarantee at least 18-24 months compatibility.
Still, like typed spans (#778), a contrib package seems fine (I'd rather have typed spans in the API than the string constants).